fixtures/room/most_booked_least_booked.py

```diff
@@ -0,0 +1,69 @@
+get_top_ten_rooms = '''
+    {
+        analyticsForMostBookedRooms(
+            startDate:"Jan 1 2019", endDate:"Jan 31 2019")
+        {
+            analytics {
+                roomName
+                meetings
+                percentage
+            }
+        }
+    }
+'''
+
+get_bottom_ten_rooms = '''
+    {
+        analyticsForLeastBookedRooms(
+            startDate:"Jan 1 2019", endDate:"Jan 31 2019")
+        {
+            analytics {
+                roomName
+                meetings
+                percentage
+            }
+        }
+    }
+'''
+
+bottom_ten_response = {
+  "data": {
+    "analyticsForLeastBookedRooms": {
+      "analytics": [
+        {
+          "meetings": 29,
+          "percentage": 100,
+          "roomName": "Entebbe",
+        }
+      ]
+    }
+  }
+}
+
+top_ten_response = {
+  "data": {
+    "analyticsForMostBookedRooms": {
+      "analytics": [
+        {
+          "meetings": 29,
+          "percentage": 100,
+          "roomName": "Entebbe",
+        }
+      ]
+    }
+  }
+}
+
+test_for_division_error = '''
```

Comment:
Great work @michael-basweti.

Please consider renaming these fixtures that you've added.
You can take a look at how other fixtures have been named for guidance.